### PR TITLE
Correct wording errors and updating message formatting 

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@smn @miltontony any other wording or formatting changes needed for bellman?

File: announce/bellman.py [hunk 1]
```diff
@@ -44,7 +44,8 @@ def people_in_group(self):
                     self.response_text = (
                         'People in ' + group_name + ':\n' + self.response_text)
             else:
-                self.response_text = 'That group doesn\'t exist'
+                self.response_text = ('The group `' + group_name +
+                                      '` doesn\'t exist')
```
Comment (smn) [inline announce/bellman.py]:
``` python
>>> group_name = "foo"
>>> "The group {0} doesn't exist".format(group_name)
"The group foo doesn't exist"
>>>
```

File: announce/bellman.py [hunk 2]
```diff
@@ -66,7 +67,7 @@ def list_my_groups(self):
                                   ' to belong to any groups. Use the '
                                   '\'opt-in\' '
                                   'command to join a group:\n'
-                                  '```/bellman opt-in GROUP_NAME```'
+                                  ' ```/bellman opt-in GROUP_NAME``` '
                                   'or use \'help\' to get more info:\n'
                                   '```/bellman help```')
```
Comment (smn) [inline announce/bellman.py]: Perhaps try double quotes so you don't have to escape all the single quotes?

File: announce/bellman.py [hunk 3]
```diff
@@ -141,8 +142,8 @@ def create(self):
                                       ' been created.\n'
                                       'To join the group, run the following '
                                       'command:\n'
-                                      '```/bellman opt-in ' + group_name
-                                      + '```')
+                                      '```/bellman opt-in ' + group_name +
+                                      '```')
```
Comment (smn) [inline announce/bellman.py]: `.format()` helps here.
Or 

``` python
("hello %s" % ("the string to insert",))
```

works as well.

**Comment by smn** [comment]
Minor stuff, :+1: regardless.

File: announce/bellman.py [hunk 4]
```diff
@@ -170,7 +171,7 @@ def announce(self):
                         self.response_text = ('The group \'' + group_name +
                                               '\' ' +
                                               'has been sent your message in'
-                                              ' the praekelt_org channel')
+                                              ' the bellman channel')
```
Comment (miltontony) [inline announce/bellman.py]: consider adding a hashtag to allow this to be clickable `the #bellman channel`

**Comment by miltontony** [comment]
:ok_hand:

**Pull request merged**